client/ayon_blender/plugins/create/create_animation.py

```diff
@@ -6,7 +6,7 @@
 class CreateAnimation(plugin.BlenderCreator):
     """Animation output for character rigs."""
 
-    identifier = "io.openpype.creators.blender.animation"
+    identifier = "io.ayon.creators.blender.animation"
```

Comment:
Coming back to [this bigger comment](https://github.com/ynput/ayon-blender/pull/115#pullrequestreview-2839828224) does updating the identifiers now mean that any newly created instance would not be detected if one _were_ to downgrade? (Not saying that's critical, but if that would not be supported (and we think that's fine) then I think we can definitely start removing anything that's trying to be supposedly backwards compatible.

Might need an opinion from @iLLiCiTiT @philippe-ynput @antirotor however.

Reply:
It does mean that indeed - however, Philippe stated that's totally ok. Downgrading to older versions introducing incompatibilities is fine if the versioning scheme at least shows that this is a bigger change, e.g. minor or major version bump.